cmd/context/update-kubeconfig.go

```diff
@@ -23,6 +23,17 @@ import (
 	oktetoLog "github.com/okteto/okteto/pkg/log"
 	"github.com/okteto/okteto/pkg/okteto"
 	"github.com/spf13/cobra"
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
+)
+
+var (
+	oktetoExecConfig = &clientcmdapi.ExecConfig{
+		APIVersion:      "client.authentication.k8s.io/v1",
+		Command:         "okteto",
+		Args:            []string{"kubetoken"},
```

Comment:
I think we should specify the `--context` and `--namespace` to avoid any interaction with the context prompt

Reply:
Agree, let's specify them so we can change k8s context without changing okteto context

Reply:
i created the func to pass the params as flags